Upstream-Status: Inappropriate [embedded specific]

lttng-modules: replace KERNELDIR with KERNEL_SRC

Since lttng-modules uses the default way of module.bbclass to
build and install lttng-modules, we do this replacement for
it as-is.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>

diff --git a/Makefile b/Makefile
index 5277386..590c8d2 100644
--- a/Makefile
+++ b/Makefile
@@ -68,19 +68,19 @@ obj-m += lib/
 endif # CONFIG_TRACEPOINTS
 
 else # KERNELRELEASE
-	KERNELDIR ?= /lib/modules/$(shell uname -r)/build
+	KERNEL_SRC ?= /lib/modules/$(shell uname -r)/build
 	PWD := $(shell pwd)
 	CFLAGS = $(EXTCFLAGS)
 
 default:
-	LTTNG_KERNELDIR=$(KERNELDIR) $(MAKE) -C $(KERNELDIR) M=$(PWD) modules
+	LTTNG_KERNELDIR=$(KERNEL_SRC) $(MAKE) -C $(KERNEL_SRC) M=$(PWD) modules
 
 modules_install:
-	LTTNG_KERNELDIR=$(KERNELDIR) $(MAKE) -C $(KERNELDIR) M=$(PWD) modules_install
+	LTTNG_KERNELDIR=$(KERNEL_SRC) $(MAKE) -C $(KERNEL_SRC) M=$(PWD) modules_install
 
 clean:
-	LTTNG_KERNELDIR=$(KERNELDIR) $(MAKE) -C $(KERNELDIR) M=$(PWD) clean
+	LTTNG_KERNELDIR=$(KERNEL_SRC) $(MAKE) -C $(KERNEL_SRC) M=$(PWD) clean
 
 %.i: %.c
-	LTTNG_KERNELDIR=$(KERNELDIR) $(MAKE) -C $(KERNELDIR) M=$(PWD) $@
+	LTTNG_KERNELDIR=$(KERNEL_SRC) $(MAKE) -C $(KERNEL_SRC) M=$(PWD) $@
 endif # KERNELRELEASE
diff --git a/probes/Makefile b/probes/Makefile
index 3272b9f..d30ca32 100644
--- a/probes/Makefile
+++ b/probes/Makefile
@@ -267,18 +267,18 @@ endif
 endif
 
 else
-	KERNELDIR ?= /lib/modules/$(shell uname -r)/build
+	KERNEL_SOURCE ?= /lib/modules/$(shell uname -r)/build
 	PWD := $(shell pwd)
 	CFLAGS = $(EXTCFLAGS)
 
 default:
-	$(MAKE) -C $(KERNELDIR) M=$(PWD) modules
+	$(MAKE) -C $(KERNEL_SRC) M=$(PWD) modules
 
 modules_install:
-	$(MAKE) -C $(KERNELDIR) M=$(PWD) modules_install
+	$(MAKE) -C $(KERNEL_SRC) M=$(PWD) modules_install
 	/sbin/depmod -a
 
 clean:
-	$(MAKE) -C $(KERNELDIR) M=$(PWD) clean
+	$(MAKE) -C $(KERNEL_SRC) M=$(PWD) clean
 
 endif
